### PR TITLE
feat(modal): component for metatags translation

### LINKED ISSUE #70: Product descriptor feature extension
At the moment the input element of the form were very small and hard to work with.

## Solution
Change from normal input to the button or clickable input element that's going to show a new popup with the text input field (Bigger space) and also explode() or replace the comma with linefeed. So it can show 
a tag/keyword per line. We also need a save button that gonna trim the input texts and implode them using comma as a delimiter

### PR DESCRIPTION
**Pull request by konthornR**
Closes #70 

1. Click left icon next to the text to open modal
image

### CODE REVIEW

File: assets/app.js [hunk 1]
```diff
@@ -68,6 +68,44 @@ var settingStorage = {
...
+            const result = this.items.filter(function (el) {
+                return el != null;
+            });
+            this.cell.row[this.field] = result.join(',');
+
+            this.save('metatags', this.cell);
+            this.close();
+        }
+
+    }
```
Comment (paibamboo) [inline assets/app.js]: indentation
image

File: assets/app.js [hunk 2]
```diff
@@ -68,6 +68,44 @@ var settingStorage = {
...
+           }
+       },
+        remove: function (index) {
+           this.items.splice(index, 1);
+        },
+        saveToDB: function () {
+            const result = this.items.filter(function (el) {
+                return el != null;
+            });
+            this.cell.row[this.field] = result.join(',');
```
Comment (lalitaCF) [inline assets/app.js:88]: Do we need to trim before add to DB?

File: assets/app.js (hunk 1 above)
Comment (konthornR) [reply assets/app.js]: data: function() && mounted: function() are the same layers with the methods: {} 

Edit: I will change the sequence to make it clearer.
image

**Comment by konthornR** [comment]
Update layout -> add header: flag and title

image

File: assets/app.js (hunk 1 above)
Comment (paibamboo) [reply assets/app.js]: ...

Comment (paibamboo) [reply assets/app.js]: image

add มันมี 3 spaces ก่อนหน้า มึงเห็นมั้ย

**Review by paibamboo — APPROVED** [review]

**Review by z3niths — APPROVED** [review]

**Pull request merged**